generators/Exercises/Grains.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Generators.Input;
+using Generators.Output;
+
+namespace Generators.Exercises
+{
+    public class Grains : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in canonicalData.Cases)
+            {
+                if (ShouldThrowException(canonicalDataCase.Expected))
+                {
+                    canonicalDataCase.ExceptionThrown = typeof(ArgumentOutOfRangeException);
+                }
+                else
+                {
+                    canonicalDataCase.Expected =  new UnescapedValue($"{canonicalDataCase.Expected}UL");
```

Comment:
Perhaps it would be better if you were to convert the expected value to an actual `ulong` instance, and then add the `UL` suffix when the value is formatted.

Reply:
Agreed, updated!